Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+public enum Colors {

```

Comment:
common mistake for enum naming - should be singular
https://mate-academy.github.io/style-guides/java/java.html#s4.8.1-enum-classes